charts/gsp-cluster/templates/02-gsp-system/http-egress-safelist.yaml

```diff
@@ -0,0 +1,71 @@
+{{- range $egressFqdn := .Values.httpEgressSafelist }}
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: {{ $.Release.Name }}-http-egress-safelist-{{ $egressFqdn.name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "gsp-cluster.name" $ }}-http-egress-safelist-{{ $egressFqdn.name }}
+    helm.sh/chart: {{ include "gsp-cluster.chart" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+spec:
+  hosts:
+  - {{ $egressFqdn.fqdn}}
+  ports:
+  - name: http
+    number: 80
+    protocol: HTTP
+  location: MESH_EXTERNAL
+  resolution: DNS
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: {{ $.Release.Name }}-http-egress-safelist-{{ $egressFqdn.name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "gsp-cluster.name" $ }}-http-egress-safelist-{{ $egressFqdn.name }}
+    helm.sh/chart: {{ include "gsp-cluster.chart" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+spec:
+  selector:
+    istio: egressgateway
+  servers:
+  - port:
+      number: 80
+      name: tls-https-egress-safelist-{{ $egressFqdn.name }}
+      protocol: HTTP
```

Comment:
can this still be istio mTLS as far as the egressgateway? (I don't pretend to understand this stuff)

Reply:
I have renamed it to http-egress instead of tls-https-egress.

Reply:
I think @philandstuff is correct; some things need changing here for mTLS to the egressgateway. I'll try to put the suggestions in and we'll see how we go.

Reply:
```suggestion
      protocol: HTTPS
```

Reply:
This has been done.